nginx/run_nginx.sh

```
@@ -240,8 +243,7 @@ create_summary_file()
 		done
 		rsec_avg=$(echo ${rsec_total}/$to_times_to_run | bc)
 		tfsec_avg=$(echo ${tfsec_total}/$to_times_to_run | bc)
-		time_file=$(grep "Requests/sec:" nginx_results_*threads_${cns}_cns.data | grep $rsec | cut -d: -f 1 | tail -1)
-
+		time_file=$(grep "Requests/sec:" nginx_results_*threads_${cns}_cns.data | cut -d: -f 1 | tail -1)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)